open-company/open-company-web
# Assets packages

Card: https://trello.com/c/GEY2hcA2

### Review with: https://github.com/belucid/open-company-infrastructure/pull/35

To test:
- deploy to staging using the `sync_images=true` setting
- deployed? good
- make sure that staging works and that the following files are loaded from the right CDN:
  - oc.js
  - oc_assets.js
  - main.css
  - all the images except of emojione.sprites.png (that is loaded always from the production CDN)
